feature-based-molecular-networking/tools/feature-based-molecular-networking/scripts/progenesis_formatter.py

```diff
@@ -41,26 +56,19 @@ def convert_to_feature_csv(input_filename, output_filename):
         output_record = {}
 
         output_record["row ID"] = str(running_scan)
```

Comment:
Why the conversion change?

Reply:
Before the script was breaking with different sample grouping made in Progenesis QI. Now it works with any number of groups columns.